public/lessons/www/anchors.md

```diff
@@ -21,20 +21,23 @@
 
 * Anchor tags create links to a linkable location
     * Within the same page, using an `#someId` selector
-    * Other page withn the site, using relative `href="/somePage.html"`
+    * Other page within the site, using relative `href="/somePage.html"`
+    * External webpages, using a full URL `href="https://www.example.com"`
 
 ```html
 <a href="#about-me">About Me</a>
 ```
 
-# Anchor Links Examples
+# Lab: Navigation
 
-[Internal Navigation Example](/lessons/responsive_layout/navigation-anchors-internal.html)
+Anchor tags are probably the most useful element to use when setting up site navigation. Let's make a page with some basic navigation capabilities.
 
-[External Navigation Example](/lessons/responsive_layout/navigation-anchors-external-0.html)
+* Create a new directory named 'example-html'
+* In html-example create a blank HTML document named `index.html`
+* Also in html-example create another HTML document named `about.html`, and put an h1 element with the text "About Page" (or something similar) in it
+* **And** an anchor tag that links back to the home page (index.html)
+* In the index.html file set up several sections with placeholder text
+  * You can easily generate placeholder text through [lorem ipsum generators](https://loremipsum.io/ultimate-list-of-lorem-ipsum-generators/)
```

Comment:
there is also the `lorem` shortcut that you can use in vscode when you are in an html document. `lorem` generates 25 words or so and then you can also add a number to the end of lorem to generate that number of words like `lorem100` to generate 100 words of lorem text.